mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -62,10 +65,22 @@ pub type OnCompactBlockFn<'a> = dyn FnMut(TonicCompactBlock) -> Result<(), MmErr
 #[allow(unused)]
 pub type OnCompactBlockFn<'a> = dyn FnMut(String) -> Result<(), MmError<UpdateBlocksCacheErr>> + Send + 'a;
 
+/// ZRpcOps trait provides asynchronous methods for performing various operations related to
+/// Zcoin blockchain and wallet synchronization.
 #[async_trait]
 pub trait ZRpcOps {
+    /// Asynchronously retrieve the current block height from the Zcoin network.
     async fn get_block_height(&mut self) -> Result<u64, MmError<UpdateBlocksCacheErr>>;
 
+    /// Asynchronously retrieve the tree state for a specific block height from the Zcoin network.
+    #[cfg(not(target_arch = "wasm32"))]
+    async fn get_tree_state(&mut self, height: u64) -> Result<TreeState, MmError<UpdateBlocksCacheErr>>;
```

Comment:
`NativeClient` doesnt support grpc client (i mean you return TreeState here, which is protobuf message), but as `get_tree_state` is in todo (in NativeClient) and we are going to use this functionality on `LightRpcClient` for now, I think its fine.